### PR TITLE
Running graphcast in real time and upload the forecasts on S3

### PR DESCRIPTION
**Pull request by SadeghTabas-NOAA**
## Description
This PR provides the functionality to run graphcast in real time with GDAS ICs. The model set up through NOAA AWS cloud to provide ML forecast results. The forecasts would be available on S3. Also, some bugs got fixed on `gdas_utility.py`. 

### Associated GraphCast Issue to close
- Closes https://github.com/NOAA-EMC/graphcast/issues/2

### Input data
- [ ] No changes are expected to input data.
- [x] Changes are expected to input data:
  - [ ] New input data.
  - [x] Updated input data.

### CODE REVIEW

File: NCEP/Readme.md [hunk 1]
```diff
@@ -11,15 +14,27 @@ The NCEP GDAS Data Processing Utility is a Python script designed to download Gl
 
 ## Overview
 
-The National Centers for Environmental Prediction (NCEP) provides GDAS data that can be used for weather prediction and analysis. This utility simplifies the process of downloading GDAS data, extracting relevant variables, and converting it into a format compatible with the GraphCast weather prediction system.
+The National Centers for Environmental Prediction (NCEP) provides GDAS data that can be used for weather prediction and analysis. This repo simplifies the process of downloading GDAS data, extracting relevant variables, and converting it into a format compatible with the GraphCast weather prediction system. In addition, it automates running GraphCast with GDAS inputs on the AWS machine.
```
Comment (junwang-noaa) [inline NCEP/Readme.md:17]: "repo" -> "repository". "AWS machine" ->"AWS cloud"

Comment (junwang-noaa) [reply NCEP/Readme.md:17]: Also, can this script writes out "zarr" format gdas data?

File: NCEP/Readme.md [hunk 2]
```diff
@@ -11,15 +14,27 @@ The NCEP GDAS Data Processing Utility is a Python script designed to download Gl
...
 
-## Prerequisites
+## Prerequisites and Installation
+
+To install the package, run:
+
+```bash
+conda create --name mlwp python=3.10
```
Comment (junwang-noaa) [inline NCEP/Readme.md:24]: I assume the prerequisites are available under ca-ufs-cpldcld project?

Comment (SadeghTabas-NOAA) [reply NCEP/Readme.md:24]: yes I did install everything on AWS cloud, just need to activate `mlwp` environment

File: NCEP/Readme.md (hunk 1 above)
Comment (SadeghTabas-NOAA) [reply NCEP/Readme.md:17]: I did not work on zarr format yet, I can work on it in next PR, that would not take so much.

File: NCEP/Readme.md (hunk 2 above)
Comment (junwang-noaa) [reply NCEP/Readme.md:24]: Perfect, thanks!

**Review by junwang-noaa — APPROVED** [review]

**Pull request merged**